modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,450 @@
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '2.4.0'
+----
+
+|`mirror.operators.catalog`
+|The Operator catalog to include in the image set.
+|String 
+Example: `registry.redhat.io/redhat/redhat-operator-index:v4.15`
+
+|`mirror.operators.full`
+|When `true`, downloads the full catalog, Operator package, or Operator channel.
+|Boolean 
+The default value is `false`.
+
+|`mirror.operators.packages`
+|The Operator packages configuration.
+|Array of objects 
+
+Example:
+[source,yaml,subs="attributes+"]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:{product-version}
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '5.2.3-31'
+----
+
+|`mirror.operators.packages.name`
+|The Operator package name to include in the image set.
+|String 
+Example: `elasticsearch-operator`
+
+|`mirror.operators.packages.channels`
+|Operator package channel configuration
+|Object
+
+|`mirror.operators.packages.channels.name`
+|The Operator channel name, unique within a package, to include in the image set.
+|String 
+Eample: `fast` or `stable-v4.15`
+
+|`mirror.operators.packages.channels.maxVersion`
+|The highest version of the Operator mirror across all channels in which it exists.
+|String 
+Example: `5.2.3-31`
+
+|`mirror.operators.packages.channels.minVersion`
+|The lowest version of the Operator to mirror across all channels in which it exists
+|String 
+Example: `5.2.3-31`
+
+|`mirror.operators.packages.maxVersion`
+|The highest version of the Operator to mirror across all channels in which it exists.
+|String 
+Example: `5.2.3-31`
+
+|`mirror.operators.packages.minVersion`
+|The lowest version of the Operator to mirror across all channels in which it exists.
+|String 
+Example: `5.2.3-31`
+
+|`mirror.operators.packages.bundles`
+|Selected bundles configuration
+|Array of objects 
+
+Example:
+[source,yaml,subs="attributes+"]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:{product-version}
+    packages:
+    - name: 3scale-operator
+      bundles:
+      - name: 3scale-operator.v0.10.0-mas
+----
+
+|`mirror.operators.packages.bundles.name`
+|Name of the bundle selected for mirror (as it appears in the catalog).
+|String 
+Example : `3scale-operator.v0.10.0-mas`
+
+|`mirror.operators.targetCatalog`
+|An alternative name and optional namespace hierarchy to mirror the referenced catalog as
+|String 
+Example: `my-namespace/my-operator-catalog`
+
+|`mirror.operators.targetCatalogSourceTemplate`
+|Path on disk for a template to use to complete catalogSource custom resource generated by oc-mirror plugin v2.
+|String 
+Example: `/tmp/catalog-source_template.yaml`
+Example of a template file: 
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v2alpha1
+kind: CatalogSource
+metadata:
+  name: discarded
+  namespace: openshift-marketplace
+spec:
+  image: discarded
+  sourceType: grpc
+  updateStrategy:
```

Comment:
A colour issue exists with this code block:

![Uploading Screenshot from 2024-06-21 12-24-26.png…]()

Reply:
Maybe add subs="attributes+ or remove extra whitespace?